libs/remix-debug/test/decoder/decodeInfo.ts

```diff
@@ -47,7 +47,7 @@ tape('solidity', function (t) {
     state = astHelper.extractStateDefinitions('test.sol:contractEnum', output.sources, null)
     stateDef = state.stateDefinitions
     parsedType = decodeInfo.parseType(stateDef[1].typeDescriptions.typeString, states, 'contractEnum', null)
-    checkDecodeInfo(st, parsedType, 1, 2, 'enum')
```

Comment:
why did the storageBytes updated here?

Reply:
as enum elements in `contractEnum` are reduced